### PR TITLE
Support for annotations on namespace level

### PR DESCRIPTION
**Pull request by fbrnc**
See #18 and updated Readme.

Example:

```yaml
apiVersion: v1
kind: Namespace
metadata:
    name: foo
    labels:
        name: foo
    annotations:
        downscaler/uptime: Mon-Sun 06:00-21:00 Europe/Berlin
```

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -16,7 +16,15 @@ Deployments are interchangeable by statefulset for this whole guide.
 
 It will scale the deployment's replicas to zero if all of the following conditions are met:
 
-* current time is not part of the "uptime" schedule (annotation ``downscaler/uptime``) or current time is part of the "downtime" schedule (``downscaler/downtime``)
+* current time is not part of the "uptime" schedule or current time is part of the "downtime" schedule. The schedules are being evaluated in following order: 
+    * ``downscaler/downtime`` annotation on the deployment/stateful set
+    * ``downscaler/uptime`` annotation on the deployment/stateful sett
```
Comment (hjacobs) [inline README.rst]: nit: typo

File: kube_downscaler/helper.py [hunk 2]
```diff
@@ -40,6 +40,6 @@ def get_kube_api():
         config = pykube.KubeConfig.from_service_account()
     except FileNotFoundError:
         # local testing
-        config = pykube.KubeConfig.from_file(os.path.expanduser('~/.kube/config'))
+        config = pykube.KubeConfig.from_file(os.environ['KUBECONFIG'] or os.path.expanduser('~/.kube/config'))
```
Comment (hjacobs) [inline kube_downscaler/helper.py]: this will throw a KeyError if `KUBECONFIG` is not set

**Comment by hjacobs** [comment]
Nice addition, thanks for the PR! See my comments.

File: kube_downscaler/helper.py [hunk 3]
```diff
@@ -40,6 +40,6 @@ def get_kube_api():
         config = pykube.KubeConfig.from_service_account()
     except FileNotFoundError:
         # local testing
-        config = pykube.KubeConfig.from_file(os.path.expanduser('~/.kube/config'))
+        config = pykube.KubeConfig.from_file((('KUBECONFIG' in os.environ) and os.environ['KUBECONFIG']) or os.path.expanduser('~/.kube/config'))
```
Comment (fbrnc) [inline kube_downscaler/helper.py]: I'm new to Python. I hope this is a good way to avoid the KeyError

Comment (hjacobs) [reply kube_downscaler/helper.py]: You can use `os.getenv()`

**Comment by hjacobs** [comment]
I'm testing this feature right now, looks good so far! Thanks :smile:

**Comment by hjacobs** [comment]
Merged/fixed and released in https://github.com/hjacobs/kube-downscaler/releases/tag/0.8

**Pull request closed without merging**